### DISCUSSION TITLE
Please take explicit responsibility for JS scrubbing of notebook previews

### DISCUSSION THREAD

**Discussion by bryevdv** [community/community#53325] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I am one of the core maintainers of Bokeh (https://github.com/bokeh/bokeh) which is an interactive data visualization library that can be used in Jupyter notebooks. Since it is intended to work in browsers, most of the actual work of Bokeh is done by the JavaScript library BokehJS. Because GitHub scrubs third-party JavaScript from all Jupyter notebook previews, Bokeh content cannot function in Jupyter notebook previews on GitHub. 

Scrubbing exernal JavaScript is a perfectly defensible position. The problem is that GitHub gives no indication to users that *it is GitHub that has taken this action*. All users see is a "broken" notebook, and then complaints splash back on Open Source maintainers, for things they have no control over. This has been ongoing for years at this point. *Please*, add an explicit and prominent disclaimer to notebook previews that makes it clear that some features may not function as expected, due to *your actions*.

**Comment by jbednar** [comment]
Exactly! I can't count how many people have come to me to say that Bokeh notebooks "don't work on Github", and assume that Bokeh has failed to do something to support Github. Indicating that Github has suppressed JS execution would help point those users towards the only people who can do anything about this situation.

Of course, even better would be if github actually did support BokehJS, which would make Github far more useful as a repository for Jupyter notebooks with rich outputs included.
